.github/workflows/sdk.yaml

```diff
@@ -200,6 +200,48 @@ jobs:
       - run: |
           cargo test --features run-showcase-tests -p integration-tests
 
+  protojson-conformance:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        rust-version: ['rust:current']
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: 'protocolbuffers/protobuf'
+          ref: '31.x'
+          path: 'protobuf'
+      - uses: actions/checkout@v4
+        with:
+          path: 'main'
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo
+          key: ${{ github.job }}-${{ runner.os }}-${{ matrix.rust-version }}-cargo-${{ hashFiles('main/Cargo.lock', 'main/.github/workflows/sdk.yaml') }}
+      - name: Setup Rust ${{ matrix.rust-version }}
+        run: |
+          set -e
+          rustup toolchain install ${{ fromJson(env.GHA_RUST_VERSIONS)[matrix.rust-version] }}
+          rustup default ${{ fromJson(env.GHA_RUST_VERSIONS)[matrix.rust-version] }}
+      - name: Display Cargo version
+        run: cargo version
+      - name: Display rustc version
+        run: rustup show active-toolchain -v
+      - name: Build conformance testee
```

Comment:
s/testee/test/

Reply:
This is the thing under test (I assume "testee" is a good name), the "test" is the thing built with Bazel that drives the testee.  I copied the nomenclature from somewhere in the ProtoJSON conformance.

I will merge, we can fix later.